perk-store-webapps/src/main/webapp/vue-app/components/PerkStoreApp.vue

```diff
@@ -532,8 +545,10 @@ export default {
     refreshProductList(selectedProductId, selectedOrderId) {
       return getProductList()
         .then((products) => {
-          if (!this.productsFilters.mine && !this.productsFilters.disabled && !this.productsFilters.soldOut) {
```

Comment:
We will have to refactor this code to paginat eresults and filter them server side.
No problem to use the same logic, but we will need to make this change later.

Reply:
noted ! to work on it later